### PR TITLE
Add my Chicago file

### LINKED ISSUE #8: cynthiaj13's checklist
- [x] Create an issue
- [x] Create branch
- [x] Create a Pull Request (PR)
- [x] Collaborate!
- [x] Merge

### PR DESCRIPTION
**Pull request by cynthiaj13**
New file that includes a few recommended places to eat in Chicago! :cityscape: :+1:

Closes #8

### CODE REVIEW

**Review by qyost — APPROVED** [review]
Looks good.

**Review by mortoland — APPROVED** [review]
Looks good

File: cynthiaj13-chicago.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Chicago, IL
+## Places to Eat
+- Real Urban BBQ in Lincolnshire near the DFS office
```
Comment (mortoland) [inline cynthiaj13-chicago.md:3]: Its ok but you need loads of napkins :cactus:

Comment (cynthiaj13) [reply cynthiaj13-chicago.md:3]: Only if you eat messily!!!

File: cynthiaj13-chicago.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # Chicago, IL
 ## Places to Eat
 - Real Urban BBQ in Lincolnshire near the DFS office
+- Marisa's Pizza near the DFS office; it's a hole in a wall place but great pizza and an italian beef special on Thursdays!
```
Comment (qyost) [inline cynthiaj13-chicago.md]: How near?

Comment (cynthiaj13) [reply cynthiaj13-chicago.md]: 2-3 miles away depending on which way you go. On Dundee just east of 294. Same strip mall area as Tong's, Butterfields Pancakes, and Goodwill :)

File: cynthiaj13-chicago.md [hunk 3]
```diff
@@ -4,3 +4,9 @@
 - Marisa's Pizza near the DFS office; it's a hole in a wall place but great pizza and an italian beef special on Thursdays!
 - Chief O'Neill's in the city for great Irish food
 - Bird's Nest in the city for wings :chicken:
+
+## Fun Things to Do
+- Cindy's Rooftop in the city on Michigan Ave for drinks and the view :cocktail:
+- K1 Speed for indoor go karting :car:
+- Revolution Brewery in the city - River North :beer:
+- Three Dots and a Dash in River North - a tiki bar for a strong and awesome :cocktail: for everyone even ships on fire!
```
Comment (qyost) [inline cynthiaj13-chicago.md:12]: How does it compare to Hala Kahiki?

**Review by qyost — APPROVED** [review]
Looks good.

File: cynthiaj13-chicago.md (hunk 3 above)
Comment (cynthiaj13) [reply cynthiaj13-chicago.md:12]: I've never been! Will have to try it out sometime! Usually bar hopping in the city lol.. :beer: :beers:

**Pull request merged**